nifi-asymmetrik-standard-bundle/nifi-asymmetrik-standard-reporting-tasks/pom.xml

```diff
@@ -64,10 +64,17 @@
             <groupId>com.yammer.metrics</groupId>
             <artifactId>metrics-core</artifactId>
         </dependency>
+        <dependency>
```

Comment:
I don't think these dependencies are needed anymore

Reply:
Removed the unneeded dependencies. (I needed the cloudwatch dependency for metric gathering)